### PR TITLE
Update the dev docs for obtaining a fit benchmark

### PR DESCRIPTION
**Pull request by robertapplin**
**Description of work.**
This PR updates the documentation on how to obtain a benchmark of fit minimizers in Mantid using the FitBenchmarking package.

**To test:**
Read through and test that the instructions work

*There is no associated issue.*

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by jhaigh0** [comment]
Would be good to have a little more detail around this
```
PYTHONPATH=/path/to/build/bin/<config> fitbenchmarking -o fitting_options.ini -p examples/benchmark_problems/Muon
```
i.e that the path is to your Mantid build, maybe just give an example path. I also feel it could be moved down to the 'Running a benchmark' section, since the instruction already says to follow the instructions in that section.

Only other thing is "Download the examples folder from..." lead me to think there would be a download link to the folder somewhere and not that I'd have to clone the repo. So some rewording there might be good.

**Review by jhaigh0 — COMMENTED** [review]
Seems good now, easy to read. :+1:

The only question I have is if the docs should make reference to 'MantidWorkbench' rather than just 'Mantid'?

**Comment by robertapplin** [comment]
> The only question I have is if the docs should make reference to 'MantidWorkbench' rather than just 'Mantid'?

Thats a good question. I think 'Mantid' makes more sense because a lot of our CurveFitting and Algorithms are in the Mantid Framework. And I think this is separate to 'MantidWorkbench' which refers more to the GUI, which uses the Framework. Mantid fitting can be used separately from the GUI in a python script.

File: dev-docs/source/ObtainingABenchmarkForMantidFitting.rst [hunk 1]
```diff
@@ -11,19 +11,38 @@ There are several scenarios in which obtaining a benchmark for the accuracy and
...
+################################
 
-Some initial setup is required on your system before we can start setting up an environment for benchmarking Mantid minimizers. Before you start this process, make sure you have a recent version of python installed, and access to git bash (which is the recommended terminal to use).
+This section will show you how to benchmark Mantid using an installed version. This is useful when you want a more realistic benchmark for fit runtimes.
+
+Initial Setup
+-------------
 
-1. Download and install the desired version of Mantid from the `downloads <https://download.mantidproject.org/>`_ page (if not already installed). Note its installation path, hereby denoted as ``[INSTALL_PATH]``.
+1. Download and install the desired version of Mantid from the `releases page <https://github.com/mantidproject/mantid/releases>`_. Note its installation path, hereby denoted as ``[INSTALL_PATH]``.
```
Comment (jhaigh0) [inline dev-docs/source/ObtainingABenchmarkForMantidFitting.rst:40]:
```suggestion
1. Download and install the desired version of MantidWorkbench from the `releases page <https://github.com/mantidproject/mantid/releases>`_. Note its installation path, hereby denoted as ``[INSTALL_PATH]``.
```
Maybe just this to change then?

**Review by jhaigh0 — CHANGES_REQUESTED** [review]

**Review by jhaigh0 — APPROVED** [review]

**Pull request merged**